src/js/actions/mask.js

```diff
@@ -142,6 +150,98 @@ define(function (require, exports) {
             menuActions.native]
     };
 
+    /**
+     * Create an circular mask the size of the currently selected layer
+     *
+     * @return {Promise}
+     */
+    var applyRectangle = function () {
```

Comment:
since they are in the mask action adding a Mask to the end of the function name seems redundant. I dont think there is much difference between Apply and Make.